<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftModelValidationComponent</class>
 <widget class="QFrame" name="CAircraftModelValidationComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model validation</string>
  </property>
  <layout class="QVBoxLayout" name="vl_AircraftModelValidation" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="lbl_Summay">
     <property name="text">
      <string>Summary will go here</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tw_CAircraftModelValidationComponent">
     <widget class="QWidget" name="tb_InvalidModels">
      <attribute name="title">
       <string>Invalid models</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_InvalidModels">
       <item>
        <widget class="BlackGui::Views::CAircraftModelView" name="tvp_InvalidModels">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Messages">
      <attribute name="title">
       <string>Messages</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Messages">
       <item>
        <widget class="BlackGui::Components::CLogComponent" name="comp_Messages">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_Buttons" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QPushButton" name="pb_TempDisableInvalid">
        <property name="text">
         <string>temp.disable invalid</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="BlackGui::Components::CSimulatorSelector" name="comp_Simulator">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_EnableStartupCheck">
        <property name="text">
         <string>check at startup</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pb_TempDisableSelected">
        <property name="text">
         <string>temp.disable selected</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_TriggerValidation">
        <property name="text">
         <string>trigger validation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_help">
        <property name="text">
         <string>help</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/aircraftmodelview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CLogComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/logcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tw_CAircraftModelValidationComponent</tabstop>
  <tabstop>tvp_InvalidModels</tabstop>
  <tabstop>cb_EnableStartupCheck</tabstop>
  <tabstop>pb_help</tabstop>
  <tabstop>pb_TriggerValidation</tabstop>
  <tabstop>pb_TempDisableInvalid</tabstop>
  <tabstop>pb_TempDisableSelected</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
